content/Why you should use GatsbyJS to build your landing page - Nicolás Santos.md

```diff
@@ -0,0 +1,26 @@
+![Gatsby](images/gatsby.png)
+
+# Why you should use Gatsby to build your landing page
+Having an awesome landing page is essential for your business. It allows your potential clients to know about what problems you solve for them and it's the visible face of your company. For that you need your page to load as fast as possible, to appear in the first places of all the major search engines, such as Google, Bing, and Yahoo, and to be available to everyone. And **Gatsby takes care of that and much more for you.**
+
+## What is Gatsby?
+Gatsby is a free and open-source framework based on React that helps developers build blazing fast **websites** and  **apps**, or that is what they say on their website.In my opinion, Gatsby is more than another framework based on React, it's the way to build your website right from the start and solve all the classical problems by integrating the plugins pre-made by the community. Also, there is an amazing community behind it, that contribute to make better and better Gatsby and create wonderful plugins that solve all types of problems with just a couple of lines of code.
+
+## What does Gatsby solve?
+Gatsby resolves a lot of things, for example, offline support, integration with WordPress, lazy loading of images and much more. But specifically, if you need to build a landing page as soon as possible, the following features stand out with Gatsby:
+
+### SEO (Search engine optimization)
+SEO is vital for your landing page, it allows your website to show on the search engine which translates in more views. With Gatsby, much of this problem is resolved as long as Gatsby generates static pages, which means the crawler of the search engine will take all the information of your page without loading the Javascript of your site, but there are other things we can do for improve this. We can add the plugin `react-helmet` for Gatsby because this is important not just for site viewers, but also for SEO — title and description metadata stored in the document head is a key component used by Google in determining placement in search results. You can see the SEO performance of your page with [Lighthouse](https://developers.google.com/web/tools/lighthouse) tools on Chrome.
+
+###  Progressive Web App
+Gatsby.js is a generator of Progressive web apps, this is a great improvement to your landing page because loads only the critical HTML, CSS, Javascript and data that you page needs to show. Also, Gatsby prefetches resources for other pages, so if you hover your mouse on a link, Gatsby preloads the page to provide a fast load. This impacts the way your users perceive the speed of your website.
```

Comment:
`this is a great improvement to your landing page because it allows you to load only the critical HTML, CSS, Javascript and data that your page needs in order to show.`